docs/resources/Audit_Log.md

```diff
@@ -29,198 +31,163 @@ Whenever an admin action is performed on the API, an entry is added to the respe
+| ------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Channel](#DOCS_RESOURCES_CHANNEL/channel-object)                                                                                                | `application_id`, `bitrate`, `default_auto_archive_duration`, `id`, `name`, `nsfw`, `permission_overwrites`, `position`, `rate_limit_per_user`, `rtc_region`, `topic`, `type`, `user_limit`, `video_quality_mode`                                                                                                                                                                                                                                                                      |
+| [Channel Overwrite](#DOCS_RESOURCES_CHANNEL/overwrite-object)                                                                                    | `id`, `type`, `allow`, `deny`                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| [Command Permission](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-permissions-object-application-command-permissions-structure)\* | snowflake                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
This seems a bit inconsistent, shouldn't the fields for `old_value` and `new_value` objects be documented in the fields column here for Command Permissions (rather than as an asterisks under the table) to fit the current style in the PR, not the special casing for the value of the `key` field which would be a better fit documented on the Audit Log Change Structure object?